azure-pipelines/release-extension.yml

```diff
@@ -138,12 +144,12 @@ extends:
                     - script: npm i -g @vscode/vsce
                       displayName: "\U0001F449 Install vsce"
 
-                    - task: AzureCLI@2
-                      displayName: "\U0001F449 Run vsce publish"
-                      condition: and(succeeded(), ${{ eq(parameters.dryRun, false) }})
-                      inputs:
-                        azureSubscription: ${{ parameters.ExtensionReleaseServiceConnection }}
-                        scriptType: pscore
-                        scriptLocation: inlineScript
-                        inlineScript: |
-                          vsce publish --azure-credential --packagePath $(vsixFileName) --manifestPath extension.manifest --signaturePath extension.signature.p7s
+                    - ${{ each publishCommand in parameters.publishCommands }}:
+                      - task: AzureCLI@2
+                        displayName: "\U0001F449 Run vsce publish"
+                        condition: and(succeeded(), ${{ eq(parameters.dryRun, false) }})
```

Comment:
It's not new to this PR, but IMO we should log the command line we would have used, when doing dry runs.